cmd/config-current.go

```diff
@@ -557,35 +558,20 @@ func (s *serverConfig) loadToCachedConfigs() {
 	}
 }
 
-// newSrvConfig - initialize a new server config, saves env parameters if
-// found, otherwise use default parameters
-func newSrvConfig(objAPI ObjectLayer) error {
-	// Initialize server config.
-	srvCfg := newServerConfig()
-
-	// Override any values from ENVs.
-	srvCfg.loadFromEnvs()
-
-	// Load values to cached global values.
-	srvCfg.loadToCachedConfigs()
```

Comment:
If you remove this ENVs will not be honored properly @krishnasrinivas you need to use this right after `loadFromEnvs()`

Reply:
The function `newSrvConfig()` itself is removed @harshavardhana 
If there is something else that you are pointing to, I am not able to get it.

Reply:
I think this comment is related to your comment at config.go:177, i have replied there.